src/core_plugins/kibana/public/doc/index.html

```diff
@@ -1,47 +1,75 @@
 <div ng-controller="doc" class="app-container">
+  <div class="kuiLocalNav">
```

Comment:
Let's get rid of the LocalNav here, since it doesn't serve its original purpose (to provide context via breadcrumbs and top-level actions) in this view. I think removing the LocalNav also helps make this feel like more of a standalone view, disconnected from any app.

We can move the document name and index pattern into the main content area using UI Framework components, and we can also simplify the markup a bit by using `kuiViewContentItem` once instead of in multiple places.

I suggest we change the markup to this:

```html
<div ng-controller="doc" class="app-container">
  <div class="kuiViewContent kuiViewContentItem">
    <h1 class="kuiTitle kuiVerticalRhythm">
      <span ng-if="!!hit">
        Document {{ hit._type }}#{{ hit._id | uriescape }}
      </span>

      <span ng-if="!hit">
        Unknown Document
      </span>
    </h1>

    <p class="kuiText kuiVerticalRhythm">
      <strong>Index pattern:</strong> {{ indexPattern }}
    </p>

    <!-- no results -->
    <div class="kuiVerticalRhythm" ng-if="status === 'notFound'">
      <div class="kuiInfoPanel kuiInfoPanel--error">
        <div class="kuiInfoPanelHeader">
          <span class="kuiInfoPanelHeader__icon kuiIcon kuiIcon--error fa-warning"></span>
          <span class="kuiInfoPanelHeader__title">
            Failed to locate document
          </span>
        </div>
        <div class="kuiInfoPanelBody">
          <div class="kuiInfoPanelBody__message">
            Unfortunately I could not find any documents matching that id, of that type, in that index. I tried really hard. I wanted it to be there. Sometimes I swear documents grow legs and just walk out of the index. Sneaky. I wish I could offer some advice here, something to make you feel better
          </div>
        </div>
      </div>
    </div>

    <!-- error -->
    <div class="kuiVerticalRhythm" ng-if="status === 'error'">
      <div class="kuiInfoPanel kuiInfoPanel--error">
        <div class="kuiInfoPanelHeader">
          <span class="kuiInfoPanelHeader__icon kuiIcon kuiIcon--error fa-warning"></span>
          <span class="kuiInfoPanelHeader__title">
            Failed to execute query
          </span>
        </div>
        <div class="kuiInfoPanelBody">
          <div class="kuiInfoPanelBody__message">
            Oh no. Something went very wrong. Its not just that I couldn't find your document, I couldn't even try. The index was missing, or the type. Go check out Elasticsearch, something isn't quite right here.
          </div>
        </div>
      </div>
    </div>

    <!-- loading -->
    <div class="kuiVerticalRhythm" ng-if="status === undefined">
      <div class="kuiPanel kuiPanel--centered">
        <div class="kuiLoadingItems">
          Loading&hellip;
        </div>
      </div>
    </div>

    <!-- result -->
    <div class="kuiVerticalRhythm" ng-if="status === 'found'">
      <doc-viewer hit="hit" index-pattern="indexPattern"></doc-viewer>
    </div>
  </div>
</div>
```

## Before (in master)

![image](https://cloud.githubusercontent.com/assets/1238659/22678002/c5a0c832-ecab-11e6-8717-fa2a368a6f66.png)

## After

![image](https://cloud.githubusercontent.com/assets/1238659/22678006/cb5c1eb6-ecab-11e6-979c-dd97390a7e78.png)